security/response-process.md

```diff
@@ -0,0 +1,254 @@
+2. GitHub security advisories.
+These are submitted similarly to GitHub issues but are private.
+3. Encrypted email to dstebila@uwaterloo.ca.
+
+When a report is received via one of the first two methods, the "on call" member of the VMT is responsible for acknowledging it promptly (the OSSF recommends within 1-2 days).
+- For email, this means responding to the reporter, cc'ing security@openquantumsafe.org.
+- For a GitHub security advisory (which will be in the "Triage" state), this means leaving a comment on the security advisory.
+
+No assessment of the report is necessary at this time.
+The purpose of the acknowledgement is to let the reporter(s) and the rest of the VMT know that the issue is being looked at.
+
+The responding VMT member assumes the role of Quarterback for the issue.
+
+Reports received via encrypted email may be more sensitive and will be handled on a case-by-case basis.
+
+> Example 1:
+A vulnerability report is received via email.
+Brian is on call.
+He replies to the reporter(s) the next day, cc'ing security@openquantumsafe.org.
+Brian is now the Quarterback for the report.
+
+> Example 2:
+A GitHub security advisory is received.
+Pravek is on call.
+He leaves a comment on the advisory the next day to acknowledge receipt of the report.
+Pravek is now the Quarterback for the report.
+
+### Internal reports
+
+Vulnerabilities may also be discovered internally by members of the OQS development team or regular contributors.
+For these reports, the Intake and Assessment phases may be streamlined at the discretion of the VMT.
+
+> Example 3:
+Michael receives an email from a former OQS developer advising him of a security issue in oqs-provider, including a patch.
+Michael immediately recognizes the issue as a vulnerability.
+Michael responds to the report, cc'ing security@openquantumsafe.org, and proceeds directly to the Patching phase.
+
+### Reports from other channels
+
+Reporters may not always follow instructions, and it is possible that security reports are received through other, non-endorsed channels (for example, email sent directly to a member of the VMT or a direct message on the PQCA discord).
+Such a report should be funnelled into one of the provided channels as the member of the VMT who first becomes aware of it sees fit.
+The focus should be on addressing the vulnerability report, not forcing the reporter(s) to resubmit via another channel.
+
+## 2. Assessment
+
+This phase begins when a vulnerability report is acknowledged by the VMT.
+It ends when the VMT responds to the reporter(s) with an assessment of the report.
+
+### Appointing a domain expert
+
+The Quarterback identifies the person or people who are most qualified to assess the report and, if necessary, develop a fix.
+The Quarterback provides them with all necessary information.
+These domain experts may not be members of the VMT, but they should be people whom the VMT trusts to handle security-related information.
```

Comment:
suggest to re-phrase "...may or may not be..." to exclude the (potential) mis-understanding that they must not be members of the VMT.